### PR TITLE
REL: rename the library

### LINKED ISSUE #3: Rename library
The name "streams" is overly generic.  It is also taken on PyPI.

We should find a better name.  One option would be to use the dask namespace and call this `dask-streams` although this makes less sense if the vanilla stream implementation is to be a core feature.

### PR DESCRIPTION
**Pull request by CJ-Wright**
The one PR to break them all.
[Cue foreboding music](https://youtu.be/l5pnoSgIuVo?t=15s)
@ordirules 
In prep for #35 
Closes #3

### CODE REVIEW

**Comment by mrocklin** [comment]
Yeah, I was thinking about this last night and was having second thoughts about `dask_streams`, mostly because no one seems to be using the dask bits yet.  Best I came up with was `streamz`.

**Comment by CJ-Wright** [comment]
I'm ok with that too.
@ordirules @danielballan

**Comment by ordirules** [comment]
great!
`streamz` might make more sense. I'm fine with either.

File: streamz/core.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from tornado.ioloop import IOLoop
 from tornado.queues import Queue
 from collections import Iterable
-from streams.graph import visualize
+from streamz.graph import visualize
```
Comment (mrocklin) [inline streamz/core.py]: Can we make this a relative import?

    from .graph import visualize

File: streamz/tests/test_core.py [hunk 2]
```diff
@@ -9,11 +9,11 @@
 from tornado.queues import Queue
 from tornado.ioloop import IOLoop
 
-import streams as s
+import streamz as s
```
Comment (mrocklin) [inline streamz/tests/test_core.py]: Perhaps start the two-letter-import-convention of `sz`?

**Comment by danielballan** [comment]
Hahahaha. I love streamz.

**Comment by CJ-Wright** [comment]
@mrocklin any more comments?

**Comment by mrocklin** [comment]
I've done a few more name changes.

**Comment by CJ-Wright** [comment]
Cool, can you merge when you are ready?

**Comment by CJ-Wright** [comment]
Awesome, thank you!

**Comment by mrocklin** [comment]
Docs are up: http://streamz.readthedocs.io/en/latest

**Comment by mrocklin** [comment]
To add to docs you just need to add to the docs/source directory.  readthedocs.org will build and publish on every push to master.  Looking at projects like dask or toolz might give good examples of docs with rst if you're not already familiar.

**Comment by CJ-Wright** [comment]
I like the docs, their like the library, minimal :laughing:

**Comment by ordirules** [comment]
great :-)

**Pull request merged**